ui/app/components/operation-status-indicator/release.hbs

```diff
@@ -0,0 +1,5 @@
+<OperationStatusIndicator @operation={{@operation}} @isDetailed={{false}}>
+  <:status-text>
+    {{if (eq @operation.status.state 1) 'Releasing' 'Released'}} on
```

Comment:
Would be nice if we can localize this. Don’t sweat it if it’s a pain though.

If you’re feeling adventurous, you could try some [advanced ICU formatting](https://formatjs.io/docs/core-concepts/icu-syntax/#select-format), though that gets pretty wacky pretty quickly 😅